geotrek/api/mobile/urls.py

```diff
@@ -4,16 +4,14 @@
 from django.conf.urls import url, include
 from rest_framework import routers
 
-from geotrek.api.mobile import views as api_mobile
 
-router = routers.DefaultRouter()
-if 'geotrek.flatpages' in settings.INSTALLED_APPS:
+if 'geotrek.flatpages' and 'geotrek.trekking' and 'geotrek.tourism' in settings.INSTALLED_APPS:
```

Comment:
Il faut répéter le in pour chaque cas car ici les deux premiers cas sont toujours considérés comme vrai.